TimberAPI/AssetLoaderSystem/AssetSystem/AssetRepository.cs

```diff
@@ -22,7 +22,7 @@ public CustomAssetBundle FindByPathAndFileName(string[] path, string fileName)
         {
             CustomAssetBundle bundle = _assetBundles.FirstOrDefault(bundle => bundle.AssetPath.SequenceEqual(path) && bundle.FileName == fileName);
             if (bundle == null)
-                throw new AssetNotFoundException();
+                throw new AssetNotFoundException(fileName);
```

Comment:
Just a sidenote, imo a cool way to throw errors in cases like these:
```c#
var bundle = _assetBundles.FirstOrDefault(bundle => bundle.AssetPath.SequenceEqual(path) && bundle.FileName == fileName) ?? throw new AssetNotFoundException(fileName);
return bundle;
```

Reply:
Prefer the chose way faster readable.
Im not sure but I thought there was something so you can skip the if statement? and throw when its null directly.
Not an needed change tho